package.json

```diff
@@ -33,12 +33,13 @@
     "is-error": "~2.2.2",
     "jsonwebtoken": "9.0.0",
     "md5": "~2.3.0",
-    "openapi-factory": "5.2.38",
+    "openapi-factory": "5.4.60",
     "retry-axios": "~2.6.0",
     "uuid": "~8.3.2"
   },
   "devDependencies": {
     "@types/jest": "^26.0.20",
+    "@types/newrelic": "^9.14.0",
```

Comment:
Soooooooooooo there's [no way to install peer dependencies with yarn](https://github.com/yarnpkg/yarn/issues/1503) in the package that requires them.

But adding types is one workaround in TS. The cleanest way would be to use NPM 7. LMK if you have opinions